foundry.toml

```
@@ -0,0 +1,9 @@
+[profile.default]
+src = "contracts"
+out = "artifacts"
+libs = ["node_modules"]
+evm_version = "shanghai"
+optimizer = true
+optimizer_runs = 2000
```

```suggestion
optimizer_runs = 200
```

should be aligned with the hardhat config I guess